addon/babel.config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "presets": [["@babel/preset-typescript"]],
+  "plugins": [
+    "@embroider/addon-dev/template-colocation-plugin",
+    [
+      "@babel/plugin-proposal-decorators", {
+        "version": "legacy"
+      }
+    ]
```

Comment:
I had an issue with `@babel/plugin-proposal-class-properties` and decided to remove it.

I had a problem with decorated class properties.
e.g. `@service('-ea-motion')  motionService!: MotionService;` was properly transpiled to decorate class property, but *in addition* I got `this.motionService = null` added to the constructor which cleaned up property value.

I assume class properties actually would be processed by babel in the consuming app at build time.

e.g. if app supports only latest browser with class properties support, having `@babel/plugin-proposal-class-properties` plugin in the addon would still transpile it to ES6 which does not seem to me as ideal behavior.

I'm 100% missing something here but not sure what.